.travis.yml

```diff
@@ -38,46 +38,23 @@ install:
   # Upgrade pip and setuptools as some times the versioning scheme hack fails
   # in Travis-CI.
   - pip install --upgrade pip setuptools
-  # Coverage dependencies, coverage for reporting coverage, pytest-cov to
-  # enable PyTest to produce coverage reports, and pytest-env to allow
-  # PyTest to use environment variables in its configuration (e.g. TZ=UTC).
-  - pip install coverage pytest-cov pytest-env pytest-runner
-  # Code style dependencies
-  - pip install pycodestyle
-  - pip install -e .
+  - pip install -e .[all]
+  - pip install coverage pytest-xdist pytest-cov pycodestyle
 
 before_script:
   - pycodestyle -v metomi/isodatetime
 
 script:
   - |
+    PYTEST_ARGS=('-n' '4')
     if "$RUN_COVERAGE"; then
-      PY_CMD="coverage run"
-      export COVERAGE_PROCESS_START="${TRAVIS_BUILD_DIR}/.coveragerc"
-      export COVERAGE_FILE="${TRAVIS_BUILD_DIR}/.coverage"
-    else
-      PY_CMD="python"
+      PYTEST_ARGS+=('--cov=metomi/isodatetime')
     fi
-    # NOTE: the way to handle long-living commands in the documentation is
-    # travis_wait 40 coverage run -m isodatetime.tests.test_time_point
-    # but it does not produce any output, which can be frustrating in case of errors.
-    # So here we are using the approach documented in the issue
-    # https://github.com/travis-ci/travis-ci/issues/4190.
-    $PY_CMD setup.py test --pytest-args='--runslow' &
-    TEST_PID="$!"
-    minutes=0
-    limit=50
-    while ps -p "${TEST_PID}" >/dev/null 2>&1; do
```

Comment:
This stuff was only needed because of the long-running monolithic test, now that its been broken apart into chunks this is no longer necessary.